core/modules/redirect/redirect.install

```diff
@@ -147,6 +147,9 @@ function redirect_update_1001() {
     db_change_field('redirect', 'language', 'langcode', $spec);
     db_add_index('redirect', 'source_langcode', array('source', 'langcode'));
   }
+  if (db_field_exists('redirect', 'status')) {
```

Comment:
Since this was a complex thing to pinpoint and explain, I think that we should add an inline comment here, pointing to the issue in our core queue. It should explain why we have added this cleanup here.

Reply:
Sure, any text suggestions?

Reply:
I'll suggest this (subject to `* ` prefix and rewrapping for 80 characters):

In D7, the status field marked a redirect as being disabled. In Backdrop, there is no such setting, so we delete all disabled redirects.
@see https://github.com/backdrop/backdrop-issues/issues/6281